src/design/brand.stories.mdx

```diff
@@ -1,16 +1,14 @@
 import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
-// Importing a `.svg` directly, results in a failed build and importing
-// the `.svg.twig` file causes linting errors (since they are not tracked).
-// Therefore, we must track a twig file to satisfy CI.
-import brandLogo from './demo/logo.twig';
+import logoSrc from '../assets/brand/logo.svg';
 
 <Meta title="Design/Brand" />
 
-# Logo
+# Brand
 
-Cloud Four logo in SVG format. The logo fill color can be modified via the CSS
-`color` property.
```

Comment:
I removed this text because it isn't accurate. It describes behavior of our Icon component, but there's nothing about the SVG asset on its own that inherits the `color` property.